pages/team-members/profile/[mail].tsx

```diff
@@ -0,0 +1,49 @@
+import { Divider, Grid, Paper } from '@mui/material';
+import CustomError from '../../../components/CustomError';
+import ProfileInfo from '../../../components/team-members/ProfileInfo';
+
+export async function getServerSideProps({ params }) {
+  const mail = String(params?.mail);
+  const res = await fetch(process.env.NEXTAUTH_URL + '/api/team-member/get-by-key?key=mail&value=' + mail);
```

Comment:
this line is too wide

Reply:
Also, why are we fetching if we are already using getServerSideProps? You can call prisma dirrectly